src/volue/mesh/_base_connection.py

```diff
@@ -229,6 +264,24 @@ def get_user_identity(self) -> core_pb2.UserIdentity:
                 be completed.
         """
 
+    @abc.abstractmethod
+    def update_external_access_token(self, access_token: str) -> None:
+        """Updates external access token used for connection to Mesh.
+
+        Args:
+            access_token: New access token to be added to authorization header
+                to each server request.
+
+        Note:
+            Does not require an open session.
+
+        Raises:
+            RuntimeError: Error message raised if the input is not valid and
+                the OAuth is not configured.
+            grpc.RpcError: Error message raised if the gRPC request could not
+                be completed.
```

Comment:
As far as I can tell there's no involved gRPC request, and no invalid input check. The only error condition is if the connection doesn't have an external access token plugin.

Reply:
Good catch. Fixed. Thanks!